### ISSUE TITLE
1.4.11 Non-Text contrast: Applicability of contrast requirements to native controls in apps

### ISSUE THREAD

**Issue by detlevhfischer** [w3c/wcag2ict#94]
In applying WCAG to native app testing, we come across the following issue:
Some native controls in iOS (the typical toggle controls) do not have the sufficient contrast of 3:1

The  SC "1.4.11 Non-text contrast" has the following exception:

> except for inactive components or where the appearance of the component is determined by the user agent and not modified by the author;

In the native app context, the equivalent of "user agent" might be construed as the respective operating system.

There is still another aspect here: in iOS settings there is a control in _Accessibility > Display & Text Size > Increase Contrast_ for users to increase the contrast generally, which means native swiches in apps would then have sufficient contrast. (The contrast of this control itself, however, is also insufficient unless activated, and many users won't be aware of the setting.)

We have verified that the insufficent color can be changed by developers. 
Currently, we are unsure whether using these system switches would mean the developer fails 1.4.11.

Should the WCAG2ICT text clarify whether the exception for user-agent determined controls is transposed to OS-determined controls in the software / native app domain? 

Any opinions?

**Comment by mitchellevan** [comment]
See #82

**Comment by detlevhfischer** [comment]
I have moved the discussion to the earlier issue #82. Closing this one.
